chain/ethereum/src/adapter.rs

```diff
@@ -24,6 +25,7 @@ use graph::{components::ethereum::EthereumNetworkIdentifier, prelude::*};
 use crate::{data_source::DataSource, Chain};
 
 pub type EventSignature = H256;
+pub type FunctionSelector = [u8; 4];
```

Comment:
I created this new type before I knew I wouldn't come back to this anymore.
I can revert those changes if necessary.

Reply:
I like it, worth a comment linking to Solidity docs on this.